modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1521,6 +1521,24 @@ public BlogsEntry updateStatus(
 				_pingTrackbacks(
 					entry, trackbacks, pingOldTrackbacks, serviceContext);
 			}
+
+			// Resources
+
+			if (GetterUtil.getBoolean(
+					serviceContext.getAttribute("isCreationScreen"))) {
```

Comment:
Can you rename the `isCreationScreen` to something more meaningful? E.g. "addResources".